core/commands/storage/upload/upload/filelist.go

```diff
@@ -0,0 +1,109 @@
+package upload
```

Comment:
if changing command to `/storage/files/list`, then this will go into `core/commands/storage/files.go`